torch_xla/csrc/helpers.cpp

```diff
@@ -619,4 +619,62 @@ xla::StatusOr<xla::XlaComputation> XlaHelpers::WrapXlaComputation(
   return builder.Build(orig_result);
 }
 
+torch::lazy::Shape XlaHelpers::ConvertXlaShapeToLazy(const xla::Shape& shape) {
```

Comment:
It looks like we already have similar utils in `TensorTypeFromXlaType`: https://github.com/pytorch/xla/blob/master/torch_xla/csrc/tensor_util.cpp#L1077.

Also, there are actually multiple similar utils in that file: `TensorTypeToRawXlaType`, `XlaTypeFromTensorType`, and `MakeXlaPrimitiveType`. @JackCaoG Any thoughts on why? and should we clean that up?

Reply:
Good catch, the existing `TensorTypeToRawXlaType` function is sufficient at least for this PR's use case. So I removed the existing `switch` case to this util function call.